lib/iris/fileformats/netcdf.py

```diff
@@ -2246,6 +2246,10 @@ def is_valid_packspec(p):
 
         conventions = CF_CONVENTIONS_VERSION
 
+        # Prioritise cube attributes conventions over the default.
+        if 'Conventions' in cube.attributes:
+            conventions = cube.attributes['Conventions']
```

Comment:
@bjlittle Sorry.  As you can tell I'm new at this.  Please tell me what I should do now.
It looks like I've got travis-ci working.
I'm guessing I should have done git pull or something.